### PR TITLE
Update to contrib 1.12.8

### PR DESCRIPTION
**Pull request by yaron2**
Updates go.mod with latest contrib.

### CODE REVIEW

**Review by ItalyPaleAle — COMMENTED** [review]
Please add release notes for the kafka fix

File: docs/release_notes/v1.12.3.md [hunk 1]
```diff
@@ -69,3 +70,49 @@ A stored procedure referenced a column named with the wrong case. This was causi
...
+Impacts users running Dapr 1.12.0-1.12.2
+
+### Root cause
+
+Dissemination did not use a background context. The logic to acquire a lock, disseminate the tables, and release the lock were all occurring in parallel.
+
+### Solution
+
+Updated the Dapr Placement service to use a background context and follow a 3 step process to acquire a lock, disseminate the tables, and release the lock.
+
```
Comment (ItalyPaleAle) [inline docs/release_notes/v1.12.3.md:101]: This is duplicate

```suggestion
```

File: docs/release_notes/v1.12.3.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ This update includes bug fixes:
 - [Fix timeouts in HTTP service invocation when resiliency policies with timeouts are applied](#fix-timeouts-in-http-service-invocation-when-resiliency-policies-with-timeouts-are-applied)
 - [Fix dissemination of placement tables](#fix-dissemination-of-placement-tables)
 - [Fix SQL Server state store not working with correctly case-sensitive collations](#fix-sql-server-state-store-not-working-correctly-with-case-sensitive-collations)
+- [Fix Kafka Pub/sub consumer bottlenecks and long startup times when subscribing to multiple topics](#fix-kafka-pub/sub-consumers)
```
Comment (ItalyPaleAle) [inline docs/release_notes/v1.12.3.md:8]: Link was broken
```suggestion
- [Fix Kafka Pub/sub consumer bottlenecks and long startup times when subscribing to multiple topics](#fix-kafka-pubsub-consumer-bottlenecks-with-multiple-topics)
```

**Review by ItalyPaleAle — APPROVED** [review]

File: docs/release_notes/v1.12.3.md (hunk 2 above)
Comment (yaron2) [reply docs/release_notes/v1.12.3.md]: Yeah forgot to save additional changes on local file before pushing

**Pull request merged**